src/test/resources/xml/IE015NormalPrelodgedDepartureDeclarationComplete.xml

```diff
@@ -81,12 +81,12 @@
 
 
         </LocationOfGoods>
-        <!-- <DepartureTransportMeans>
+       <DepartureTransportMeans>
             <sequenceNumber>1</sequenceNumber>
             <typeOfIdentification>40</typeOfIdentification>
             <identificationNumber>LKW 52/145-6978-569/AF362:1</identificationNumber>
```

Comment:
Can we change this value for identification number with something that would pass our validation, please :)